libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -266,15 +267,29 @@ func (d *addDockerMetadata) getProcessCgroups(pid int) (map[string]string, error
 }
 
 // getContainerIDFromCgroups checks all of the processes' paths to see if any
-// of them are associated with Docker. Docker uses /docker/<CID> when
-// naming cgroups and we use this to determine the container ID. If no container
-// ID is found then an empty string is returned.
-func getContainerIDFromCgroups(cgroups map[string]string) string {
+// of them are associated with Docker. Docker uses /docker/<CID> and
+// K8s uses /kubepods/<class>/<podID>/<cID> when naming cgroups and
```

Comment:
When you say K8s uses `/kubepods/<class>/<podID>/<cID>` when naming cgroups is that even when K8s is running with Docker, or only in the case its running with a different CSI?

Reply:
Right, I should specify. It works like that when K8s is running with Docker as a container runtime. I can't verify for other cases like CRI-O or containerd